Manylinux build crashes due to python syntax error in wrap

Hi @varunagrawal and @ProfFan , I think some recent wrap changes broke the wheel-builder in `gtsam-manylinux`. Here's the error message:

```
Scanning dependencies of target pybind_wrap_gtsam
[  7%] Building C object gtsam/3rdparty/metis/libmetis/CMakeFiles/metis-gtsam.dir/__/GKlib/itemsets.c.o
[  7%] Generating gtsam.cpp
[  7%] Building C object gtsam/3rdparty/metis/libmetis/CMakeFiles/metis-gtsam.dir/__/GKlib/mcore.c.o
Traceback (most recent call last):
  File "/gtsam/wrap/scripts/pybind_wrap.py", line 13, in <module>
    import gtwrap.interface_parser as parser
  File "/gtsam/wrap/gtwrap/interface_parser/__init__.py", line 17, in <module>
    from .classes import *
  File "/gtsam/wrap/gtwrap/interface_parser/classes.py", line 18, in <module>
    from .function import ArgumentList, ReturnType
  File "/gtsam/wrap/gtwrap/interface_parser/function.py", line 58
    self.parent: Union[ArgumentList, None] = None
               ^
SyntaxError: invalid syntax
make[2]: *** [python/gtsam.cpp] Error 1
make[1]: *** [python/CMakeFiles/pybind_wrap_gtsam.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

Could yall take a quick look when you get a chance?

Link to run:
https://github.com/borglab/gtsam-manylinux-build/runs/2802606418?check_suite_focus=true

Link to all runs:
https://github.com/borglab/gtsam-manylinux-build/actions/runs/928431417

--- a/gtwrap/interface_parser/function.py
+++ b/gtwrap/interface_parser/function.py
@@ -47,7 +47,7 @@ class Argument:
             self.ctype = ctype
         self.name = name
         self.default = default
-        self.parent: Union[ArgumentList, None] = None
+        self.parent = None  # type: Union[ArgumentList, None]
 
     def __repr__(self) -> str:
         return self.to_cpp()

